File: api/store/model/product.go
package model

type Product struct {
	Id                int     `json:"id"`
	ProductCategoryId int     `json:"productCategoryId"`
	Name              string  `json:"name"`
	Description       string  `json:"description"`
	Price             float64 `json:"price"`
	Deposit           float64 `json:"deposit"`
	Duration          float64 `json:"duration"`
}

File: front-end/src/api/models/product.ts
export interface Product {
    id: number;
    productCategoryId: number;
    name: string;
    description: string;
    price: number;
    deposit: number;
    duration: number;
}

export const emptyProduct = (): Product => {
    return {
        id: 0,
        productCategoryId: 0,
        name: '',
        description: '',
        price: 0,
        deposit: 0,
        duration: 0,
    };
};
